TypeCobol.Analysis.Test/BasicCfgInstrs/SearchPerform0.dot

```diff
@@ -0,0 +1,90 @@
+﻿digraph Cfg {
+compound=true;
+node [
+shape = "record"
+]
+
+edge [
+arrowtail = "empty"
+]
+Block0 [
+label = "{START|}"
+]
+Block1 [
+label = "{Block1|    PERFORM SMPL-SEARCH\l}"
+]
+subgraph cluster_1{
+color = blue;
+label = "SMPL-SEARCH";
+Block16 [
+label = "{SMPL-SEARCH. Block16|    DISPLAY 'SEARCH EXAMPLE....'\l    SET SEQ TO 1\l}"
+]
+Block17 [
+label = "{Block17|    SEARCH SUBJECT\l}"
```

Comment:
Why Block17 not directly included in  Block16 (as in Start0 Block15 contains the START instruction) ?

Reply:
Block17 has to be independent so as to have the returning edge from Block18. This is specific to SEARCH statement which acts as a loop.